File: backend/components/metiers/threed/utils/helpers/mocks/mock_helper.py
# mock_helper.py – Mock ultra avancé helpers (clé en main)
def mock_validation_data():
    """
    Mock de validation pour tests RGPD, audit, accessibilité
    :return: dict
    """
    from datetime import datetime
    return {
        'id': 1,
        'user': 'mocked',
        'email': 'mock@example.com',
        'consent': True,
        'timestamp': datetime.utcnow().isoformat()
    }

File: backend/components/metiers/threed/utils/helpers/mocks/mock_helper.js
// mock_helper.js – Mock ultra avancé helpers (clé en main)
/**
 * Mock de validation pour tests RGPD, audit, accessibilité
 * @returns {object}
 */
function mockValidationData() {
  return {
    id: 1,
    user: 'mocked',
    email: 'mock@example.com',
    consent: true,
    timestamp: new Date().toISOString()
  };
}
module.exports = { mockValidationData };
